testsuite/forge/src/backend/local/swarm.rs

```diff
@@ -201,6 +201,14 @@ impl LocalSwarm {
                 Ok((validator.peer_id(), public_network))
             })
             .collect::<Result<HashMap<_, _>>>()?;
+
+        // We print out the root key to make it easy for users to deploy a local faucet
+        let encoded_root_key = hex::encode_upper(&root_key.to_bytes());
```

Comment:
lol why upper, I think we use lower for everything else :P

Reply:
Aah, this was copy pasta from other code. I can update it 😄

Reply:
Here's where I copied it from: https://github.com/aptos-labs/aptos-core/blob/c11321e7337e9c1469ecba38f808d1187192b9b3/crates/aptos/src/common/types.rs#L397